cross-chain/base/hardhat.config.ts

```diff
@@ -0,0 +1,151 @@
+import type { HardhatUserConfig } from "hardhat/config"
+
+import "@nomiclabs/hardhat-etherscan"
+import "@keep-network/hardhat-helpers"
+import "@nomiclabs/hardhat-waffle"
+import "hardhat-gas-reporter"
+import "hardhat-contract-sizer"
+import "hardhat-deploy"
+import "@typechain/hardhat"
+import "hardhat-dependency-compiler"
+
+const config: HardhatUserConfig = {
+  solidity: {
+    compilers: [
+      {
+        version: "0.8.17",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 1000,
+          },
+        },
+      },
+    ],
+  },
+
+  paths: {
+    artifacts: "./build",
+  },
+
+  networks: {
+    hardhat: {
+      deploy: [
+        // "deploy_l1",
+        "deploy_l2",
+      ],
+    },
+    goerli: {
+      url: process.env.L1_CHAIN_API_URL || "",
+      chainId: 5,
+      deploy: ["deploy_l1"],
+      accounts: process.env.L1_ACCOUNTS_PRIVATE_KEYS
+        ? process.env.L1_ACCOUNTS_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["etherscan"],
+    },
+    mainnet: {
+      url: process.env.L1_CHAIN_API_URL || "",
+      chainId: 1,
+      deploy: ["deploy_l1"],
+      accounts: process.env.L1_ACCOUNTS_PRIVATE_KEYS
+        ? process.env.L1_ACCOUNTS_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["etherscan"],
+    },
+    baseGoerli: {
+      url: process.env.L2_CHAIN_API_URL || "",
+      chainId: 84531,
+      deploy: ["deploy_l2"],
+      accounts: process.env.L2_ACCOUNTS_PRIVATE_KEYS
+        ? process.env.L2_ACCOUNTS_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["basescan"],
+      gasPrice: 1000000000,
+      // companionNetworks: {
+      //   l1: "goerli",
+      // },
+    },
+    base: {
+      url: process.env.L2_CHAIN_API_URL || "",
+      chainId: 8453,
+      deploy: ["deploy_l2"],
+      accounts: process.env.L2_ACCOUNTS_PRIVATE_KEYS
+        ? process.env.L2_ACCOUNTS_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["basescan"],
+      gasPrice: 1000000000,
```

Comment:
Why do we need to set the `gasPrice` property?

Reply:
For some reason, it threw an error without this property when I tried to deploy on Goerli. Each chain has its own unique nuances, and Base needs this property.

Reply:
This is strange. I was able to run most of the deployment transactions with `gasPrice` commented out. One of the transactions failed with `ProviderError: transaction underpriced`. I commented out this line for now. In case we face the problem we will add it back and align the value to the value needed for the current state of the chain.